pkg/kubelet/rkt/rkt.go

```diff
@@ -2142,10 +2196,26 @@ func (r *Runtime) GetPodStatus(uid kubetypes.UID, name, namespace string) (*kube
 		}
 	}
 
-	// TODO(euank): this will not work in host networking mode
-	containerID := kubecontainer.ContainerID{ID: string(uid)}
-	if status, err := r.networkPlugin.GetPodNetworkStatus(namespace, name, containerID); err == nil {
-		podStatus.IP = status.IP.String()
+	// If we are running no-op network plugin, then get the pod IP from the rkt pod status.
+	if r.networkPlugin.Name() == network.DefaultPluginName {
```

Comment:
This means that a host-network pod with a network plugin will still run `GetPodNetworkStatus` below even though we don't have a namespace, right?
That doesn't seem like the right behavior.